module/VuFind/config/module.config.php

```diff
@@ -814,74 +832,190 @@
 
 // Define static routes -- Controller/Action strings
 $staticRoutes = [
-    'Alphabrowse/Home', 'Author/FacetList', 'Author/Home', 'Author/Search',
-    'Authority/FacetList', 'Authority/Home', 'Authority/Search',
-    'Blender/Advanced', 'Blender/Home', 'Blender/Results',
-    'Browse/Author', 'Browse/Dewey', 'Browse/Era', 'Browse/Genre', 'Browse/Home',
-    'Browse/LCC', 'Browse/Region', 'Browse/Tag', 'Browse/Topic', 'Cart/doExport',
-    'BrowZine/Home', 'BrowZine/Search',
-    'Cart/Email', 'Cart/Export', 'Cart/Home', 'Cart/MyResearchBulk',
-    'Cart/Processor', 'Cart/Save', 'Cart/SearchResultsBulk',
-    'Channels/Home', 'Channels/Record', 'Channels/Search',
-    'Checkouts/History', 'Checkouts/PurgeHistory',
-    'Comments/DeleteComments', 'Comments/UserList',
+    'Alphabrowse/Home',
+    'Author/FacetList',
+    'Author/Home',
+    'Author/Search',
+    'Authority/FacetList',
+    'Authority/Home',
+    'Authority/Search',
+    'Blender/Advanced',
+    'Blender/Home',
+    'Blender/Results',
+    'Browse/Author',
+    'Browse/Dewey',
+    'Browse/Era',
+    'Browse/Genre',
+    'Browse/Home',
+    'Browse/LCC',
+    'Browse/Region',
+    'Browse/Tag',
+    'Browse/Topic',
+    'BrowZine/Home',
+    'BrowZine/Search',
+        'Cart/doExport',
```

Comment:
Extra indentation here for some reason:

```suggestion
    'Cart/doExport',
```